TESTING.md

```diff
@@ -84,7 +80,6 @@ Tasks for a tester to verify when approving a patch. Use complex window layouts
 ### Window Titles
 
 - [ ] Disabling window titles in Firefox works (Check debian and flatpak packages)
-  - This may make firefox windows move around randomly when toggled, and you may have to restart firefox for it to work properly, but it should work besides that.
```

Comment:
This is still the case, and may be a good thing to leave in here so that testers immediately know this is not a regression.